### PR TITLE
Feature ux work

### PR DESCRIPTION
**Pull request by alexdivadi**
Made a couple changes:

Only update scriptures with remoteConfig if number of scriptures = 0
Return user back to the list they were on after adding a scripture

### CODE REVIEW

File: integration_test/app_test.dart [hunk 1]
```diff
@@ -59,4 +80,21 @@ Future<void> enterNameToEdit(WidgetTester tester, String newName) async {
   expect(ok, findsOneWidget);
   expect(cancel, findsOneWidget);
 
+}
+
+Future<void> enterVerseToAdd(WidgetTester tester, String reference, String collection) async {
+
```
Comment (neiljaywarner) [inline integration_test/app_test.dart:86]: nice

**Comment by neiljaywarner** [comment]
@alexdivadi thanks for working on this!
let's discuss the zero change please or something.
i am pretty sure if there's a database with scriptures but no "comfort verses" collection existing with verses locally then it should be added. will this code allow this?

**Comment by alexdivadi** [comment]
The problem for me is if I rename the list, it will pull it from remote
config again each time I open the app.

I think the remote collections should only be pulled at the user's request
except for the initial population of the db

**Comment by neiljaywarner** [comment]
Let's find a best of both worlds please.
Decent solution might be to retain"name from remote" as a field.

If new remote collections can only be added on first install, it makes the
remote feature almost useless.

In my opinion

The point is to be able to reach existing users and add things dynamically.

To build the feature with two collections and add eight more later and it
doesn't have to be a developer adding them and doesn't have to have a new
release

In fact, it becomes relatively easy to have a completely separate list for
your users in Korea or Washington.


....sent from my phone

**Comment by neiljaywarner** [comment]
@alexdivadi this is great, what about the color change from orange to something else, did i miss that? thanks

**Comment by neiljaywarner** [comment]
oops my bad i see the purple.

**Pull request merged**